CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # http-signature-zcap-verify ChangeLog
 
+## 6.0.0 -
+
+### Changed
+- **BREAKING**: Use `@digitalbazaar/zcapld@3.0` instead of `ocapld@v2.0`.
```

Comment:
Please say a few word about why this manifests as a breaking change here and/or reference ocap/zcap changelog entry.

Reply:
One of the breaking changes is that zcapld: Only support Node.js >=12.

Which means that the engine in package.json needs to be updated here and we need to drop support for node <12 here.